src/cl_sii/dte/parse.py

```diff
@@ -570,16 +570,17 @@ def _validate_rut(v: object) -> object:
     return v
 
 
-class _DteXmlReferenciaParser(pydantic.v1.BaseModel):
+class _DteXmlReferenciaParser(pydantic.BaseModel):
     """
     Parser for ``/Documento/Referencia``.
     """
 
-    class Config:
-        allow_mutation = False
-        anystr_strip_whitespace = True
-        arbitrary_types_allowed = True
-        extra = pydantic.v1.Extra.forbid
+    model_config = pydantic.ConfigDict(
+        frozen=True,
+        str_strip_whitespace=True,
+        arbitrary_types_allowed=True,
+        extra='forbid',
+    )
```

Comment:
⚠️ This review comment applies to all similar cases.

Please sort arguments lexicographically.